core/dbt/contracts/graph/node_args.py

```diff
@@ -16,4 +17,5 @@ class ModelNodeArgs:
     version: Optional[NodeVersion] = None
     latest_version: Optional[NodeVersion] = None
     deprecation_date: Optional[datetime] = None
+    access: Optional[str] = AccessType.Protected.value
```

Comment:
I opted for a string instead of AccessType to have a simpler interface for the external node construction. Core will still raises an error for an invalid value when it's converted it to an AccessType in `ModelNode.from_args`

Reply:
Makes sense. We do a similar thing for enums in unparsed classses.